File: tools_C++/src/Procedures.cpp
/*
 * Procedures.cpp
 *
 *  Created on: Apr 12, 2017
 *      Author: broihier
 */

#include "Procedures.h"

Procedures::Procedures(std::string category): DatabaseComponent(category+".testDbProcedures") {
}

Procedures::~Procedures() {
}

const std::string& Procedures::getProcedureText(const std::string& id) const{
	return getComponentText(id);
}

File: tools_java/tools/src/tools/Procedures.java
package tools;
/**
 * 
 * @author broihier
 * <pre>
 * Class for creating Procedures objects
 * </pre>
 */
public class Procedures extends DatabaseComponent {
	/** 
	 * Constructor for creating Procedures objects
	 * 
	 * @param category name that maps to 'category'.testDbProcedures files
	 * 
	 */
	public Procedures(String category) {
		super(category+".testDbProcedures");
	}
	/**
	 * Method to get the procedure text for a particular testID
	 * @param testID test case ID
	 * @return getComponentText(testID)
	 */
	public String getProcedureText(String testID) {
		return getComponentText(testID);
	}
}
